sdbus_async/bluez/profile_api.py

```diff
@@ -48,3 +48,34 @@ async def unregister_profile(
         profile_path: str,
     ) -> None:
         raise NotImplementedError
+
+class ProfileInterfaceAsync(
```

Comment:
Hmm. Should there be a blocking version of this interface? It looks like it is intended to mostly be used by Bluez itself to call different D-Bus clients. (unless you want to manually disconnect something without bluez)

Reply:
I find that sdbus_block/ directory is empty, so blocking version of bluez interfaces are all missing. Not only blocking Profile interface, other many blocking interfaces are to be added :) It will be another future work.

My current understanding is that the Profile interface is required for applications to establish connection with legacy (not BLE, Bluetooth Low Energy) Bluetooth device. So far, async version of Profile interface succeeds to establish connection with my Bluetooth device. It looks enough for me at this moment.